### ISSUE TITLE
Os environment should contain network information, and manpage should be updated

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#420]
Originally reported of Google Code with ID 364.

```
Currently the os environment passes NIC_n_NETWORK as the network name.
It should also pass:
NIC_n_NETWORK_{UUID,SUBNET,GATEWAY,SUBNET6,GATEWAY6,etc.}
also perhaps NETWORK should be renamed to NETWORK_NAME (but that's less important, and can be left for backwards compatibility with beta1)

The os api interface manpage should also be updated to mention all network related variables

Thanks,

Guido
```
Originally added on 2013-02-07 13:12:38 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I'll handle this one.
```
Originally added on 2013-02-07 14:31:42 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in commit d89168f

Thanks,

Guido
```
Originally added on 2013-02-08 16:39:45 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
